hapi-utils/pom.xml

```diff
@@ -13,6 +13,19 @@
     <version>0.14.0-SNAPSHOT</version>
   </parent>
 
+  <repositories>
+    <repository>
+      <id>sdk-alpha</id>
+      <url>https://oss.sonatype.org/content/repositories/comswirlds-1100</url>
```

Comment:
@qnswirlds Based on Richard's suggestion on friday's scrum, we might need to merge official sdk instead of alpha sdk to master.

Reply:
I think we need `0.9.4-alpha.2` for some tests and features. The staging repo should be available for a while until we have the official SDK from the platform team. Btw do you know the timeline of them releasing this official SDK?

Reply:
@qnswirlds  Nosh just informed that Nathan is working on it